<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyDialog</class>
 <widget class="QDialog" name="KeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify Shortcut Key</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 12px &quot;Noto Sans&quot;, &quot;FreeSans&quot;;
	background: #1A1A1A;
	color: whitesmoke;
}

/* QLineEdit */

QLineEdit {
	background: black;
	border: 1px solid #454A4D;
	border-radius: 2px;
	padding: 2px;
}
QLineEdit:focus {
	border: 1px solid #008098;
}

/* QPushButton */

QPushButton {
	background: #454A4D;
	border: 1px solid #454A4D;
	border-bottom: 1px solid black;
	border-radius: 2px;
	color: whitesmoke;
	min-width: 76px;
	min-height: 17px;
	padding: 4px 6px;
}
QPushButton:focus {
	border: 1px solid #008098;
	outline: none;
}
QPushButton:pressed {
	background: #2E3133;
}
QPushButton:disabled {
	background: transparent;
	border: 1px solid #454A4D;
	color: #B8B8B8;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QKeySequenceEdit" name="keySequenceEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="commandLineEdit">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="labelLineEdit">
       <property name="placeholderText">
        <string>(optional)</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Shortcut:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Command:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Label:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>C&amp;lear Shorcut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeButton">
       <property name="text">
        <string>Chang&amp;e</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>clearButton</tabstop>
  <tabstop>changeButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
